lib/flipper/test_help.rb

```diff
@@ -0,0 +1,29 @@
+module Flipper
+  module TestHelp
+    extend self
+
+    def before_all
+      Flipper.configure do |config|
+        config.adapter { Flipper::Adapters::Memory.new }
+        config.default { Flipper.new(config.adapter) }
```

Comment:
Does this apply all the `config.use` adapters? It would seem like no. Should it? I know everyone would then have the strict issue but I wonder if it would be confusing if someone uses `use` and doesn't see it in their adapter stack.

Reply:
Yes. `config.adapter` will return the storage adapter wrapped in `use` adapters.